core/src/Platforms/Android/SystemWebview/AuthenticationActivity.cs

```diff
@@ -127,15 +127,23 @@ protected override void OnResume()
                 string chromePackage = GetChromePackage();
                 if (string.IsNullOrEmpty(chromePackage))
                 {
-                    throw new MsalClientException(MsalClientException.ChromeNotInstalledError,
-                        "Chrome is not installed on the device, cannot proceed with authentication");
+                    const string chromeNotInstalledMessage = " Chrome is not installed on the device, cannot proceed with authentication";
```

Comment:
>const string chromeNotInstalledMessage =  [](start = 20, length = 41)

I was thinking to place this in the same or a similar location of the ID. Alternatively near the Actual factory method.